src/App.js

```diff
@@ -1,26 +1,45 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter as Router, Route, Link, Switch } from 'react-router-dom'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    <Router>
+      <Link to="/">Home</Link>
+      <Link to="/list">List</Link>
+      <Link to="/add">Add</Link>
+      <Switch>
+        <Route exact path="/">
+          <Home />
+        </Route>
+        <Route exact path="/list">
+          <List />
+        </Route>
+        <Route exact path="/add">
+          <Add />
+        </Route>
+      </Switch>
+    </Router>
+  )
+}
+
```

Comment:
We think each of these functions should be separate components imported into the Router component.